content/2017-09-12_python-komunita-a-ja.md

```diff
@@ -0,0 +1,63 @@
+
+Moje původní role zakladatele a organizátora brněnského Pyva se brzy rozšířila v toho, kdo propojuje lidi, tmelí je dohromady, a ví o všem, co se kde děje. Díky tomu, že jsem se nebál sociálních sítí, znal všechny ze všech měst a zároveň na sebe upoutával pozornost i lidí mimo komunitu, začal jsem být ideální osobou, která by měla propagovat vše, co se v komunitě dělo. Sám jsem se pak ještě pojmenoval jako "maskot české Python komunity", abych dal najevo, že nestojím za vším, co propaguji, ale že je to často tvrdá práce mnoha dalších lidí.
+
+Na brněnské Pyvo jsem pravidelně jezdil několik let i potom, co jsem se přestěhoval do Prahy. Vloni jsem ale začal slýchat pozdravy jako "jé, co že jsi přijel?" a já si uvědomil, že jezdím spíš nepravidelně než pravidelně. Po tom, co se mi povedlo vyvolat zmatek a konat vánoční Pyvo na dvou místech zároveň, jsem se z jeho organizace potichu vytratil. A stávající organizátoři dál organizují, navíc se přidal Jirka Psotka a sehnal nový prostor. Pyvo existuje dál. Což je naprosto skvělé! Když jsem v dubnu 2011 v Brně srazy založil, nejspíš by mě nenapadlo, že po šesti letech budou stále existovat. A to, že dokážou existovat i beze mě a jít si vlastní cestou, je pro mě jako zakladatele největší odměnou.
+
+Python komunita roste a mění se. Na našem čistě organizátorském Slacku je přes 120 lidí. Mnoho z nich už nejspíš ani neví, že nějaký Honza Javorek existuje. Vznikají nové aktivity, o kterých už jen tuším, že existují. A s každým dalším kurzem, sprintem nebo workshopem počet zapojených lidí stoupá.
+
+Všechno je to rozhodně správný trend. Historicky jsme nikdy nefungovali centralizovaně, ale spíš federačně. Tedy tak, že každá úderná skupina byla samostatná v konání i rozhodování, ale vystupovalo se společně pod jednou vlajkou a probíhala určitá synchronizace a výměna znalostí. Tedy existovali lidé, jako je Petr Viktorin nebo já, kteří se účastnili tolika aktivit v komunitě, že vlastně zajišťovali ono informační a kulturní propojení.
+
+Jenže soustředění mnoha informací do mozku pár osob neškáluje a nemůže prospívat nikomu ze zúčastněných. Je potřeba zpřehlednit a posílit federační způsob, jakým pracujeme. Je potřeba, aby skupin mohlo být ještě víc a dokázaly fungovat suverénněji. Existence implicitních předpokladů v hlavách pár osob místo explicitně zdokumentovaných návodů brání lidem v orientaci, rozletu, a tomu, aby jsme byli všichni na podobné vlně. Je potřeba víc vysvětlovat, víc dokumentovat, víc sepisovat jaké jsou naše hodnoty a jak věci děláme, jak o nich rozhodujeme, jak o nich diskutujeme.
+
+Přemýšlel jsem tedy o tom, že by se měla moje role změnit a jasněji určit. Abych věděl, na čem přesně mám pracovat, a co mám pustit z hlavy. Aby i ostatní měli jasno v tom, co ode mě mají očekávat, a co už ne. Takže tady to je. Snažil jsem se před programováním apod. činnostmi, kde jsem nahraditelný, upřednostnit věci, kde mohu uplatnit svůj přehled v komunitě a znalost její historie.
+
+## Čím se chci zabývat
```

Comment:
Pozor, celá tahle část je vyšinutá z větné vazby, koukej, jak to nefunguje:

 * Chci se zabývat pomocník Martina Bílka....
 * Chci se zabývat majitel domén...
 * ...

Nebudu to měnit, protože nevím, kterým směrem, bys to chtěl měnit.

Nabízím:

 * chci se zabývat pomáháním Martinovi Bílkovi, převodem domén, tvorbou nové verze, zadáváním, psaním (nadpis zůstává, body se rapidně mění)

nebo

 * chci být pomocníkem, majitelm, správcem, autorem (nadpis se mění, body se mění v pádu)